src/macaron/slsa_analyzer/git_url.py

```diff
@@ -362,7 +362,7 @@ def get_repo_name_from_url(url: str) -> str:
 def get_repo_full_name_from_url(url: str) -> str:
     """Extract the full name of the repository from the remote url.
 
-    The full name is in the form <owner>/<name>.
+    The full name is in the form <owner>/<name>. Note that this function assume `url` is a remote url.
```

Comment:
```suggestion
   The full name is in the form <owner>/<name>. Note that this function assumes `url` is a remote url.
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/153/commits/bdeba388db589ff62f060dbb87c5512bb05ba587